ui/dashboards/src/utils/copy.ts

```diff
@@ -0,0 +1,47 @@
+// Copyright 2022 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
```

Comment:
This is very nitpicky, so feel free to ignore it if you disagree, but I wonder if something like "messaging" or "messages" would be less confusing than "copy," since copy has multiple meanings (e.g. copy & paste).

Reply:
You had a question about where to put this. It feels a little weird in `utils` to me with it just being a collection of constants with no utility functions. I wonder if it would make slightly more sense in `models`? Curious what other folks think on that.

Reply:
Do we want to start using constants for all labels/texts? Then I think it will make more sense to create a folder separate from `models` for these labels/texts that are constants

Reply:
Yay thank you for thinking this through with me. 

Good call about `utils` and `models` and maybe a new folder ... it's not exactly a util, because it's not a function... not really a model either since models describe the structure of data...

Maybe I should call this file `constants.ts` and either leave it in the `utils` folder? Or call it `tooltip.ts` and move into a new folder called `constants`? I don't love love either of these, but will keep thinking it through... I also like the other name suggestions for `copy` since `copy` was confusing. 

One thing I am cautious about -- I wouldn't advocate yet for moving to a process where we move all text to constants that live in a centralized file -- mainly because I've been in a codebase with that kind of rule, and it created a lot of overhead. I only want to add overhead when we see benefits (right now, the benefit I'm hoping for is that all these tooltips have a consistent tone)

Reply:
I like the idea of a `constants` directory with a `tooltipMessaging.ts` or similarly named file to communicate what it's for.

Reply:
OK sweet -- 
* created `constants` directory
* renamed `copy.ts` to `user-interface-text.ts`
* renamed constants `TOOLTIP_COPY` and `ARIA_LABEL_COPY` to  `TOOLTIP_TEXT` and `ARIA_LABEL_TEXT`

I like this much better 🎉